code/modules/modular_computers/file_system/programs/generic/file_browser.dm

```diff
@@ -55,18 +55,24 @@
 			choices[FS.name] = FS
 		var/file_source = input(usr, "Choose a storage medium to use:", "Select Storage Medium") as null|anything in choices
 		if(file_source)
-			current_filesource = choices[file_source]
```

Comment:
Deliberate removal?

Reply:
Yes, this has been moved beneath both the if and else clause so that the filesource doesn't change to an errored network if the checks don't pass.